cwms-data-api/src/main/java/cwms/cda/ApiServlet.java

```diff
@@ -308,8 +285,10 @@ protected void configureRoutes() {
                 new ParametersController(metrics), requiredRoles);
         cdaCrud("/timezones/{zone}",
                 new TimeZoneController(metrics), requiredRoles);
+        LevelsController levelsController = new LevelsController(metrics);
         cdaCrud("/levels/{" + Controllers.LEVEL_ID + "}",
-                new LevelsController(metrics), requiredRoles);
+                levelsController, requiredRoles);
+        get("/levels/timeseries/{level-id}", levelsController::getLevelAsTimeSeries);
```

Comment:
I feel like this should be "/levels/{level-id}/timeseries/"

Reply:
basic logic there is it's the "time series for the level" not the "level of a timeseries."

Reply:
That's what I had wanted to do, but I was following the /timeseries/recent/{group-id} pattern

Reply:
Hmm. arguably that timeseries should be `/timeseries/{group-id}/recent` instead of the way it is.

Reply:
Sample size of one doesn't really make a pattern, so I'm happy to do levels/{level-id}/timeseries

Reply:
True. Make the change, what you originally intended flows more naturally which is what we should do if at all possible.